src/app/file-manager/services/gcs-api.service.ts

```diff
@@ -22,4 +24,10 @@ export class GcsApiService extends GcsService {
     return this.http.get(url);
   }
 
+  public uploadFiles(bucketName: String, files: any[]) {
+      if (files !== null && files.length > 0) {
```

Comment:
How can we get here if these fields are missing? And if we do, how do we report an error?

Reply:
It should never happen because we have a validation before to execute this method, on file explorer upload component, the button Upload Selection must be disable if files were not selected